thirdparty/libwebp/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+project(libwebp)
+cmake_minimum_required(VERSION 3.5.1)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules")
+include("koreader_thirdparty_common")
+include("koreader_thirdparty_git")
+
+enable_language(C CXX)
+
+assert_var_defined(CC)
+assert_var_defined(CXX)
+assert_var_defined(CPPFLAGS)
+assert_var_defined(LDFLAGS)
+assert_var_defined(CHOST)
+
+ep_get_source_dir(SOURCE_DIR)
+ep_get_binary_dir(BINARY_DIR)
+
+set(CFG_ENV_VAR "CC=\"${CC}\" CXX=\"${CXX}\" CPPFLAGS=\"${CPPFLAGS}\" LDFLAGS=\"${LDFLAGS}\" ")
+# libwebpdecoder.so is enough to decode single frame webp images. To decode animated webp,
+# we'd need the twice larger libwebp.so and libwebpdemux.so.
```

Comment:
I assume that comment is outdated, since you did go the extra mile ;).

Reply:
Yes, but it's a comment on the next commented line.
Actually, I originally went with libwebp.so. Then, so the thing about `--enable-libwebpdecoder` in configure --help, and thought "oh, smaller lib ! let's go with that" - removed all the libwebp stuff, build, tested - and realized if failed on an animated webp. So, put the initial stuff back.
So, let's keep it as a kind of documentation of why we can't go with libwebpdecoder.so :)

Reply:
Oh, yeah, definitely, I just sort of implied it only needed a tiny tense tweak (say, something like "we need the full set of libs").